ConsistencyManagerTests/ListenerTests.swift

```diff
@@ -117,6 +117,7 @@ class ListenerTests: ConsistencyManagerTestCase {
                     listener = strongListener
 
                     addListener(strongListener, toConsistencyManager: consistencyManager)
+                    waitOnDispatchQueue(consistencyManager)
```

Comment:
Yeah, I don't have a great memory for this test, but this seems like a decent fix to me.

Reply:
I will work more on it, this seems to always be happening, it was not being caught because the test was running only for macOS device, the other devices were just building the project but not running the test. My change makes it build and test for each device.

Reply:
Does it fail locally? And if so, on what device? I'm also happy to help take a look.

Reply:
If I increase it from 100 to 2000 it somehow happens locally:
<img width="573" alt="image" src="https://user-images.githubusercontent.com/103670/72270969-05927c80-35f4-11ea-92b8-e4ab568341b3.png">

But then it is a `EXC_BAD_ACCESS`
<img width="992" alt="image" src="https://user-images.githubusercontent.com/103670/72270909-ea277180-35f3-11ea-83f5-585909a1fba6.png">